### PR TITLE
*/*: pkg-config fixes (respect ${PKG_CONFIG}, add missing dips on it, improve patches)

### PR DESCRIPTION
**Pull request by thesamesam**

### CODE REVIEW

File: app-admin/apache-tools/apache-tools-2.4.46.ebuild [hunk 1]
```diff
@@ -25,6 +25,8 @@ DEPEND="${RDEPEND}
 	sys-devel/libtool"
 BDEPEND="virtual/pkgconfig"
 
+BDEPEND="virtual/pkgconfig"
```
Comment (ionenwks) [inline app-admin/apache-tools/apache-tools-2.4.46.ebuild:28]: Isn't there another BDEPEND="virtual/pkgconfig" just above?

File: app-admin/apache-tools/apache-tools-2.4.47.ebuild [hunk 2]
```diff
@@ -25,6 +25,8 @@ DEPEND="${RDEPEND}
 	sys-devel/libtool"
 BDEPEND="virtual/pkgconfig"
 
+BDEPEND="virtual/pkgconfig"
```
Comment (ionenwks) [inline app-admin/apache-tools/apache-tools-2.4.47.ebuild:28]: here too

File: app-i18n/canna/canna-3.7_p3-r3.ebuild [hunk 3]
```diff
@@ -21,6 +21,7 @@ RDEPEND="canuum? (
 		sys-libs/ncurses:=
 	)"
 DEPEND="${RDEPEND}
+	virtual/pkgconfig
 	x11-misc/gccmakedep
 	x11-misc/imake
 	canuum? ( virtual/pkgconfig )
```
Comment (ionenwks) [inline app-i18n/canna/canna-3.7_p3-r3.ebuild:27]: pkgconfig twice?

File: dev-lang/swig/swig-4.0.2.ebuild [hunk 4]
```diff
@@ -16,11 +16,11 @@ IUSE="ccache doc pcre"
 RESTRICT="test"
 
 RDEPEND="
-	virtual/pkgconfig
 	pcre? ( dev-libs/libpcre )
 	ccache? ( sys-libs/zlib )
 "
 DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
```
Comment (ionenwks) [inline dev-lang/swig/swig-4.0.2.ebuild:23]: Are you sure this is right? swig does runtime compilation stuff, RDEPEND pkgconfig could make sense

**Review by ionenwks — COMMENTED** [review]
Not much to say, only had a quick look after coming across the PR.

File: dev-lang/swig/swig-4.0.2.ebuild (hunk 4 above)
Comment (thesamesam) [reply dev-lang/swig/swig-4.0.2.ebuild:23]: Yeah, seems fine.

File: dev-libs/libzia/libzia-4.25.ebuild [hunk 5]
```diff
@@ -19,8 +19,9 @@ RDEPEND="dev-libs/glib:2
 	media-libs/libpng:0
 	ftdi? ( dev-embedded/libftdi:1 )"
 
-DEPEND="${RDEPEND}
-	virtual/pkgconfig"
+DEPEND="${RDEPEND}"
+
```
Comment (SoapGentoo) [inline dev-libs/libzia/libzia-4.25.ebuild:23]: spacing but w/e

File: media-sound/tapestrea/tapestrea-0.1.0.5-r1.ebuild [hunk 6]
```diff
@@ -26,6 +26,8 @@ DEPEND="${RDEPEND}
 	sys-devel/bison:0
 	sys-devel/flex:0"
```
Comment (SoapGentoo) [inline media-sound/tapestrea/tapestrea-0.1.0.5-r1.ebuild:27]: fix these too and the spacing afterwards?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**